examples/templates/vue/README.md

```diff
@@ -0,0 +1,61 @@
+# {{ YOUR TITLE GOES HERE }}
+
+## Description
+
+{{ YOUR DESCRIPTION GOES HERE }}
+
+## Live preview
+
+You can preview this template on CodeSandbox  under the URL: https://githubbox.com/handsontable/code-snippets/tree/master/examples/templates/vue
+
+## How to run this project
+
+### Installation
+
+Call `npm install` to install all necessary dependencies. This command will first remove `node_modules` folder and install testing tools (`jasmine`, `jasmine-console-reporter`, `puppeteer` and `http-server`) from the root directory.
+
+### Development
+
+To start local development server call `npm run start`. Now you can visit http://localhost:8080 to view this project.
+
+### Testing
+
+In order to run tests for this project you have two options:
+
+1. Call `npm run start` which will launch the development server and after the server is running, call `npm run jasmine` to run test specs.
+2. Call `npm run test` which will build the project, then start the server on the production output folder, run test specs and close the server.
+
+## License
```

Comment:
I guess my mistake here, too detailed. We should use simple MIT:

```## Licensing

This wrapper is released under the MIT license.

Copyrights belong to Handsoncode sp. z o.o.
```


Applies to all README.md's

Reply:
Done https://github.com/handsontable/handsontable/pull/7515/commits/4cedf4a98e61e6ca3bd794aacadbac53b0f563fa

Reply:
Yeah, but you left the "wrapper" wording. Shouldn't it be "This example is release under the MIT license." ?

Reply:
Yup.. done https://github.com/handsontable/handsontable/pull/7515/commits/f2451dc5d3e984c26d4069587dcb03b2d6517b2a